### ISSUE TITLE
Where do you load VGG weights?

### ISSUE THREAD

**Issue by tegillis** [Aaronphilip2/Image-Segmentation-for-Faces#1]
I don't see where you loaded the pretrained VGG-Face weights. Is the accuracy you reported from training from scratch? Thanks.

**Comment by aaronphilip** [comment]
I decided not to add the code for transforming the pretrained weights I used into a Keras compatible format since that code is specific to those weights. However I can add an option to load  in pretrained weights that have already been formatted in an h5 file like this https://keras.io/getting-started/faq/#how-can-i-save-a-keras-model.

**Comment by tegillis** [comment]
Okay, and do you train the entire network after loading the weights or just fine tune the second half?

**Comment by aaronphilip** [comment]
The weights were just for the VGG-16 part of the model. I froze that portion of the model and only trained the second half that does the upsampling.

**Comment by tegillis** [comment]
Thanks
